js/packages/openinference-vercel/src/VercelSpanProcessor.ts

```diff
@@ -0,0 +1,26 @@
+import { ReadableSpan, SpanProcessor } from "@opentelemetry/sdk-trace-base";
+import { safelyGetOpenInferenceAttributes } from "./utils";
+import { ReadWriteSpan } from "./types";
+
+export class VercelSpanProcessor implements SpanProcessor {
```

Comment:
I guess I sorta change my mind - sorry. This is really an OpenInferenceSpanProcessor - it's exported from openinference-vercel and by that nature it's specific to vercel I think

Reply:
Yeah that makes sense to me